index.html

```diff
@@ -1672,11 +1677,11 @@ <h3>
         <p>
           If the user agent honors the value of the <code>theme_color</code>
           member as the <a>default theme color</a>, then that color serves as
-          the <a>theme color</a> for all browsing contexts to which the
-          manifest is <a>applied</a>. However, a document may override the
-          <a>default theme color</a> through the inclusion of a valid [[HTML]]
-          <code>meta</code> element whose <code>name</code> attribute is
-          "theme-color".
+          the <a>theme color</a> for all browsing contexts <a data-lt=
+          "within-scope-manifest">within scope</a> to which the manifest is
+          <a>applied</a>. However, a document may override the <a>default theme
```

Comment:
Should that be MAY @marcoscaceres ?

Reply:
As I wrote on the other review comment: you can't use the word "may" without it being in all-caps, making it a normative requirement. And you can't impose a normative requirement on the document; all requirements are imposed on the user agent.

(I know this is a problem with the existing text as well. We should fix it.)